app/utils/package.scala

```diff
@@ -37,14 +36,17 @@ package object utils {
   private def normalise(nino: Nino): Nino =
     Nino(normaliseNino(nino.nino))
 
-  def areEqual(source: Nino, target: Nino) =
+  def areEqual(source: Nino, target: Nino): Boolean =
     normalise(source) == normalise(target)
 
-  def getSid(request: Request[_]) =
+  def getSid(request: Request[_]): String =
     request.session.get(SessionKeys.sessionId).getOrElse("")
 
   def isPtaJourneyUseOnlyOnAuthorisedPage(implicit request: Request[_]): Boolean =
     request.cookies.get(ApplicationConfig.TAMC_JOURNEY).exists { _.value == ApplicationConfig.TAMC_JOURNEY_PTA } ||
-      !request.cookies.get(ApplicationConfig.TAMC_JOURNEY).isDefined
+      request.cookies.get(ApplicationConfig.TAMC_JOURNEY).isEmpty
 
+  def scottishResident(request: Request[_]): Boolean = {
```

Comment:
As this is a boolean method it should really start with "is", i.e. isScottishResident